test/helpers/kubectl.go

```diff
@@ -227,6 +233,12 @@ func CreateKubectl(vmName string, log *logrus.Entry) (k *Kubectl) {
 	return k
 }
 
+// EnsureLabels labels nodes in vagrant env. If you are running tests on different cluster you need to label your nodes by yourself
+func (kub *Kubectl) EnsureLabels() {
```

Comment:
The how-to-test docs are still a todo for me :/ Once I write them I'll mention this. (but perhaps joestringer already wrote some of this. I don't think it came up for him, though, since it was a single-node test)